assets/xml/objects/object_stk2.xml

```diff
@@ -1,86 +1,105 @@
 ﻿<Root>
+    <!-- Most of Skull Kid's animations, as well as assets for his curse effects and the Ocarina in the Clock Tower. -->
     <File Name="object_stk2" Segment="6">
-        <Animation Name="object_stk2_Anim_00055C" Offset="0x55C" />
+        <!-- Skull Kid Animations -->
+        <Animation Name="gSkullKidCurseStartAnim" Offset="0x55C" /> <!-- Original name is "kubi_katakata" ("neck clack/rattle") -->
         <!-- <Blob Name="object_stk2_Blob_00056C" Size="0x24" Offset="0x56C" /> -->
```

Comment:
They have a very specific pattern, but one unlike anything I've ever seen. Check this one out for example:
```c
u8 object_stk2_unaccounted_00056C[] = {
    0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 
    0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x00, 
    0x00, 0x00, 0x00, 0x00, 
};
```

And this one as well:
```c
u8 object_stk2_unaccounted_00131C[] = {
    0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 
    0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 
    0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 
    0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 
    0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 0x00, 0x11, 
    0x00, 0x00, 0x00, 0x00, 
};
```
They seem to just be some combination of 0x00 and 0x11s. But I've never quite seen anything like that before; do you know what this could represent?

Reply:
That does not look familiar, no, open to suggestions.